sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -1438,15 +1438,10 @@ export class CertificateClient {
     let result: UpdateCertificateResponse;
 
     try {
-      result = await this.client.updateCertificate(
```

Comment:
This was a linting update.

Reply:
can we do the linting/formatting in a separate PR to reduce noise?

Reply:
I will separate it tomorrow 👍🏻